### ISSUE TITLE
fix existing bug with the Alarm Manager and UnifiedPush connector

### ISSUE THREAD

**Issue by nucleus-ffm** [nucleus-ffm/foss_warn#137]
We need to use the Alarm Manager in combination with the UnifiedPush Connector. Unfortunately, there is still an open bug that leads to incompatibility. We need to find the bug and create a fix for the plugin to be able to use both

**Comment by PureTryOut** [comment]
Wait, why do we need both UnifiedPush and polling through AlarmManager? The whole point of a push service like UnifiedPush is that you don't need to manually wake up and check for things, you'll automatically get a notification instead 🤔

**Comment by nucleus-ffm** [comment]
> The whole point of a push service like UnifiedPush is that you don't need to manually wake up and check for things, you'll automatically get a notification instead

Yes, if the push service works as expected. I see two reasons why we might still want to periodically wake the app. 

1. Update the subscription. We can not expect the user to open the app once every week or get a notification that also wakes up the app. So we have to reliably wake the app, e.g. once a day or even less, to update the subscription.
2. Check if the push notification system still works. Imagine, e.g., someone uses a self-hosted unified push server, and the server goes down. It seems to be a good idea to check if the system still works. We may not need the AlarmManager for that. The WorkManager would probably be enough.

The bug is described here https://github.com/UnifiedPush/flutter-connector/issues/123

**Comment by nucleus-ffm** [comment]
migrated discussion to codeberg https://codeberg.org/UnifiedPush/flutter-connector/issues/28
